core/src/dird/dird.cc

```diff
@@ -397,8 +397,6 @@ int main(int argc, char* argv[])
 
   if (test_config) {
     if (my_config->HasDeprecationWarnings()) {
-      // FIXME: cannot use Jmsg here as only M_FATAL and M_ERROR_TERM will
```

Comment:
A comment would be helpful anyway to understand why none of the Dmg or Jms can be used.

Reply:
:+1: